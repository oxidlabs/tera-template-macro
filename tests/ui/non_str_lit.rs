use tera_template_macro::TeraTemplate;

#[derive(TeraTemplate, serde::Serialize)]
#[template(path = 69)]
struct Smoke {
    hello: String
}

fn main() {}