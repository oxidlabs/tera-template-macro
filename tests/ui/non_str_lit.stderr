error: The assignment to `path` must be a string literal.
 --> tests/ui/non_str_lit.rs:4:19
  |
4 | #[template(path = 69)]
  |                   ^^
